docs/docs/monitoring/analytics.mdx

```diff
@@ -1,19 +1,876 @@
+      dialect: "postgresql"
+      url: "localhost"
+      db: "tracker"
+      username: postgres
+      password: password
+
+  event_broker:
+      type: kafka
+      topic: rasa-events
+      url: localhost:29092
+      partition_by_sender: true
+```
+
+:::note
+
+Running manual ingestion of past events multiple times will result in
+duplicated events. There is currently no deduplication implemented in
+Analytics. Every ingested event will be stored in the database,
+even if it was processed previously.
 :::
 
+### Connect a BI Solution
+
+Connecting a business intelligence platform to the data warehouse varies for
+each platform. We provide example instructions for Metabase and Tableau but
+you can use any BI platform which supports AWS Redshift or PostgreSQL.
+
+#### Example: Connecting Metabase
+
+Metabase is a free and open-source business intelligence platform. It
+provides a simple interface to query and visualize data. Metabase can
+be connected to PostgreSQL or Redshift databases.
+
+- [Connecting Metabase to PostgreSQL](https://www.metabase.com/data_sources/postgresql)
+- [Connecting Metabase to Redshift](https://www.metabase.com/data_sources/amazon-redshift)
+
+#### Example: Connecting Tableau
+
+Tableau is a business intelligence platform. It provides a flexible interface
+to build business intelligence dashboards. Tableau can be connected to
+PostgreSQL or Redshift databases.
+
+- [Connecting Tableau to PostgreSQL](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm)
+- [Connecting Tableau to Redshift](https://help.tableau.com/current/pro/desktop/en-us/examples_amazonredshift.htm)
+
+## Example Queries
+
+Inspiration on how to analyze the conversations your assistant is having. The
+example queries are created using Metabase.
+
+### Daily Chatbot Users
+
+#### Query using SQL
+
+```sql
+SELECT CAST(rasa_sender.first_seen AS date) AS "Date", COUNT(*) AS "Number of Users"
+FROM rasa_sender
+GROUP BY CAST(rasa_sender.first_seen AS date)
+ORDER BY CAST(rasa_sender.first_seen AS date) ASC
+```
+
+#### Using the Metabase Query Builder
+
+<div align="center">
+  <img
+    alt="Example query for a query showcasing the number of users per day."
+    src={useBaseUrl("/img/analytics/number-of-users-query.png")}
+    width="100%"
+  />
+</div>
+
+#### Result
+
+<div align="center">
+  <img
+    alt="Example result for a query showcasing the number of users per day."
+    src={useBaseUrl("/img/analytics/number-of-users-result.png")}
+    width="100%"
+  />
+</div>
+
+## Data Structure Reference
+
+Descriptions and examples of the data structure created by Analytics.
+The data structure is created by the Analytics pipeline and treated as
+a public API. [Internal tables](#internal-tables) should be considered private and may change
+without notice.
+
+Table overview:
+
+<div align="center">
+  <img
+    alt="An overview of the components of Rasa Pro."
+    src={useBaseUrl("/img/analytics/analytics-db-schema.png")}
```

Comment:
can we comment here in the docs how this file can be generated? it'll avoid future questions on Slack on how to update it ;)